.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
@LorenzMaehringer please revert the changes here. We don’t want these issue templates as parts of our PR